netpyne_ui/experiments.py

```diff
@@ -169,25 +212,29 @@ def _parse_experiment(directory: str) -> model.Experiment:
 
     :raises ExperimentsError
     """
-    path = os.path.join(constants.NETPYNE_WORKDIR_PATH, constants.EXPERIMENTS_FOLDER, directory)
+    path = os.path.join(
+        constants.NETPYNE_WORKDIR_PATH, constants.EXPERIMENTS_FOLDER, directory
+    )
 
     try:
-        with open(os.path.join(path, EXPERIMENT_FILE), 'r') as f:
+        with open(os.path.join(path, EXPERIMENT_FILE), "r") as f:
```

Comment:
What's the point about changing single quotes to double quotes?

Reply:
That's the black formatter, switched to VSCode and have now different setup

Reply:
Not really a fan of this kind of formatting rules but not against if used to keep prs clean, but this is quite the opposite. As for the frontend, it would be better to have one chore pr with all the formatting rules applied in one shot rather than taking the linting burden into history.

Reply:
yes that formatting change slipped in accidentally. The only formatter that is available in VSCode for which I don't see formatting changes without further configuration in the current codebase is `none` :) , so I'm now switching to none until we settled on one format and applied it to the complete backend codebase in one commit.

Reply:
the pep8 formatter is available, and it's less disruptive than black (the code is mosty pep8 compliant and it's mostly about lines and spaces)